Sources/Sentry/SentryTimeToDisplayTracker.m

```diff
@@ -137,6 +137,22 @@ - (void)reportFullyDisplayed
     [_dispatchQueueWrapper dispatchAsyncOnMainQueue:^{ self->_fullyDisplayedReported = YES; }];
 }
 
+- (void)finishSpansIfNotFinished
+{
+    if (self.initialDisplaySpan.isFinished == NO) {
+        [self.initialDisplaySpan finish];
+    }
+
+    if (self.fullDisplaySpan.isFinished == NO) {
+        SENTRY_LOG_WARN(
+            @"You didn't call SentrySDK.reportFullyDisplayed(). Finishing full display span with "
+            @"status deadline exceeded.");
+
+        [self.fullDisplaySpan finishWithStatus:kSentrySpanStatusDeadlineExceeded];
```

Comment:
m: I just dont know if this is a good status for this case.

Reply:
I also thought about that. It's the same as when you forget to call reportFullyDisplayed, and the 30s timeout hits. I wanted to keep it consistend.